Find a functionally equivalent code:android.media.MediaPlayer.setAuxEffectSendLevel:COMMENT
<android.media.MediaPlayer: void setAuxEffectSendLevel(float)>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Sets the send level of the player to the attached auxiliary effect
 * {@see #attachAuxEffect(int)}. The level value range is 0 to 1.0.
 * <p>By default the send level is 0, so even if an effect is attached to the player
 * this method must be called for the effect to be applied.
 * <p>Note that the passed level value is a raw scalar. UI controls should be scaled
 * logarithmically: the gain applied by audio framework ranges from -72dB to 0dB,
 * so an appropriate conversion from linear UI input x to level is:
 * x == 0 -> level = 0
 * 0 < x <= R -> level = 10^(72*(x-R)/20/R)
 * @param level send level scalar
 */

@@ -1 +1,6 @@
-
+{
+    if (isRestricted()) {
+        return;
+    }
+    _setAuxEffectSendLevel(level);
+}

